core/src/main/java/org/apache/iceberg/rest/RESTSessionCatalog.java

```diff
@@ -106,7 +106,7 @@ public class RESTSessionCatalog extends BaseSessionCatalog
   private ResourcePaths paths = null;
   private Object conf = null;
   private FileIO io = null;
-  private MetricsReporter reporter = null;
+  private MetricsReporter customMetricsReporter = null;
```

Comment:
Why change the name? Does it matter how the metrics reporter was created?

Reply:
I was confused for a sec when reading the code I wrote a while ago and thought why this wasn't called `customMetricsReporter` in the first place, hence the change. I can change it back if you think it's not worth renaming this